main.py

```diff
@@ -1,10 +1,52 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as read_file:
+        players = json.load(read_file)
+
+    for player_name, info in players.items():
+        if not Race.objects.filter(
+                name=info["race"]["name"]).exists():
+            Race.objects.create(
+                name=info["race"]["name"],
+                description=info["race"]["description"]
+            )
+        race = Race.objects.get(
+            name=info["race"]["name"])
+
+        if info["guild"] is not None:
+            if not Guild.objects.filter(
+                    name=info["guild"]["name"]).exists():
+                guild = Guild.objects.create(
+                    name=info["guild"]["name"],
+                    description=info["guild"]["description"]
+                )
+            guild = Guild.objects.get(
+                name=info["guild"]["name"])
+        else:
+            guild = None
```

Comment:
Move it before if and not use else)

Reply:
It will not pass one test if I do like this without else: 
        guild = None
        if info["guild"] is not None:
            if not Guild.objects.filter(
                    name=info["guild"]["name"]).exists():
                guild, created = Guild.objects.get_or_create(
                    name=info["guild"]["name"],
                    description=info["guild"]["description"]
                )

Reply:
@ArsenPidhoretskyi also I have some questions: 
1) In models Skill and Player we have the same pieces of code (FK) "race = models.ForeignKey(Race, on_delete=models.CASCADE)". Can we write it like a link? For e.g. in Player model write "race = Skill.race"? 
2) Before pushing, I needed to delete db (checklist rule). But when I want to see how it works I'd like to have db again, but when I run the script without db, I have errors. To avoid this situation, I just need to do db manual (create file, than create field e.t.c.) or we have another quicker option?

Reply:
Why it not pass the test? It is so strange because the code will be the same) But okay)

Reply:
1. No, because it is a little different. But, as for me, it is also strange)
Race has multiple skills. And race has multiple players

Reply:
2. No need to delete db, just not commit it. You can add db to gitignore if you need. Use python manage.py migrate it will auto create

Reply:
![image](https://user-images.githubusercontent.com/113930309/216818969-ad7e08ae-b59b-4627-9e1f-d58a58c8a08e.png)
here scrin with this test

Reply:
Probably the problem was in .exist method row. Without this, it will work normal